100.en.adoc

```diff
@@ -2,116 +2,118 @@
 
-In contrast to classical taxonomy, which is almost an entirely manual (human) process, clustering DNA sequences into taxonomic concepts uses algorithmic analysis of similarity and other signals (such as phylogenetic and probability), as well as some human editing. The resulting OTUs vary in stability, presence of reference sequences and physical material, alignments and cut-off values, and OTU identifiers (such as DOIs). Even more importantly, they vary in scale, from local study- or project-specific libraries to global databases that enable broader cross-study comparison. Unlike centralized and codified of Linnaean taxa that are formally described in research publications, OTUs live in the evolving digital reference libraries that differ by taxonomic focus, barcode genes and other factors.
+In contrast to classical taxonomy, which is a heavily manual (human) process, clustering DNA sequences into taxonomic concepts relies on algorithmic analysis of similarity and other signals (such as phylogeny and probability), as well as some human editing. The resulting OTUs vary in stability, presence of reference sequences and physical material, alignments and cut-off values, and OTU identifiers (such as DOIs). Even more importantly, they vary in scale, from local study- or project-specific libraries to global databases that enable broader cross-study comparison. In contrast to the centralization and codification of Linnaean taxa that are formally described in research publications, OTUs are distributed across multiple evolving digital reference libraries that differ in taxonomic focus, barcode genes and other factors.
 
-Algorithms for taxonomic annotation of eDNA will typically assign each unique sequence to their nearest taxonomic group in a reference set, given some relatedness and confidence criteria. For poorly known groups of organisms, such as prokaryotes and fungi, this may be a non-Linnean placeholder name for a (cluster-based) taxon, and this taxon will often be ranked above species level. No reference database contains all species in a given group, unlike what many users seem to think: this misunderstanding is a source of numerous taxonomic misidentifications during the last 30 years.
+Algorithms for taxonomic annotation of eDNA will typically assign each unique sequence to the nearest taxonomic group in a reference set, based on some criteria for relatedness and confidence. For poorly known groups of organisms, such as prokaryotes and fungi, the annotation may be a non-Linnean placeholder name for a (cluster-based) taxon, and this taxon will often be ranked above species level. No reference database contains all species in a given group. Many users neglect this fact, which has been the source for numerous taxonomic misidentifications during the last 30 years.
```

Comment:
Users' neglect of this fact has been the source of numerous taxonomic misidentifications during the last 30 years.